Recolor.php

```diff
@@ -20,7 +20,10 @@ class Recolor {
     /**
      * Process an image file
      *
+     * @param string $imageName Which image to process
+     * @param string $newFilename New filename to save to
```

Comment:
I'd accidentally missed these out! If you're not yet on PHP 7 with scalar type hints, it's important to document what inputs your method is expecting.